src/db.ls

```diff
@@ -67,16 +67,28 @@
     db.DB = {}
     minimatch = require \minimatch
     try
-      db.DB = JSON.parse do
-        require \fs .readFileSync "#dataDir/dump.json" \utf8
-      console.log "==> Restored previous session from JSON file"
+      db.DB = {}
+      fs.readdirSync "#dataDir/dump/" .forEach (f) ->
+        Object.assign do
+          db.DB,
+          JSON.parse fs.readFileSync "#dataDir/dump/#f"
+      console.log "==> Restored previous session from JSON files"
       db.DB = {} if db.DB is true
     Commands =
       bgsave: (cb) ->
-        fs.writeFileSync do
-          "#dataDir/dump.json"
-          JSON.stringify db.DB,,2
-          \utf8
+        if !fs.existsSync "#dataDir/dump/"
+          fs.mkdirSync("#dataDir/dump/")
+        sheets = {}
+        for k, v of db.DB
+          id = k.split("-").pop!.split("_")[0]
+          if !sheets[id]
+            sheets[id] = {}
+          sheets[id][k] = v
+        for k, v of sheets
```

Comment:
This will save all sheets irrespective of which changed since the last save, which is wasteful.